### PR TITLE
Updated US BLS (2017Q1)

### LINKED ISSUE #349: Update Data: US Bureau of Labor Statistics
The US Bureau of Labor Statistics (QCEW) has released data through 2017Q1. Ours is currently only up to 2015Q4. 

This has been requested by a user

### PR DESCRIPTION
**Pull request by antoniocarlon**
- Added data up to 2017Q1
 - Improved time management to make it easier to add new quarters/years

Closes #349

### CODE REVIEW

File: tasks/us/bls.py [hunk 1]
```diff
@@ -1,17 +1,21 @@
...
+from tasks.us.census.tiger import GeoidColumns
 
 from collections import OrderedDict
 from luigi import IntParameter, Parameter, WrapperTask
 
 import os
+import glob
+
+
+MAX_TIMESPAN = '2017Q1'  # Example '2017Q1'
```
Comment (amartinj) [inline tasks/us/bls.py]: Pass this as an argument to the method using it, so it's configurable from the classes using this one

File: tasks/us/bls.py [hunk 2]
```diff
@@ -265,23 +273,9 @@ def populate(self):
 class AllQCEW(WrapperTask):
 
     def requires(self):
-        for year in range(2012, 2016):
+        maxyear, maxqtr = [int(n) for n in MAX_TIMESPAN.split('Q')]
+        for year in range(2012, maxyear + 1):
             for qtr in range(1, 5):
+                if year == maxyear and qtr > maxqtr:
```
Comment (amartinj) [inline tasks/us/bls.py]: I would do the opposite condition:
```
if !(year == maxyear and qtr > maxqtr):
 yield QCEW(year=year, qtr=qtr)
```

File: tasks/us/bls.py (hunk 1 above)
Comment (antoniocarlon) [reply tasks/us/bls.py]: Totally agree

File: tasks/us/bls.py (hunk 2 above)
Comment (antoniocarlon) [reply tasks/us/bls.py]: I prefer applying some Bool algebra and ellaborate it a bit:
```
if year < maxyear or qtr <= maxqtr:
    yield QCEW(year=year, qtr=qtr)
```

**Comment by amartinj** [comment]
Moving to acceptance! 💪

**Pull request merged**